inductiva/resources/machines_base.py

```diff
@@ -47,8 +47,14 @@ def __init__(self,
         if machine_type not in inductiva.resources.list_available_machines():
             raise ValueError("Machine type not supported")
 
+        if disk_size_gb < 0 or disk_size_gb > 100:
+            raise ValueError(
+                "Disk size must be a positive value smaller than 100 GB")
+
         self.machine_type = machine_type
         self.disk_size_gb = disk_size_gb
+        self._true_disk_size_gb =\
```

Comment:
Save this variable and add the one for the `data_disk_gb`, since this is a property of the user's dedicated machines.